lib/ui/views/home/home_viewmodel.dart

```diff
@@ -300,6 +296,10 @@ class HomeViewModel extends BaseViewModel {
     return _managerAPI.getLatestManagerReleaseTime();
   }
 
+  Future<String?> getLatestManagerVersion() {
```

Comment:
What is this for?

Reply:
unintentional test code, thought it was marked out with // but i guess i forgor 💀 
ill remove it completely